libr/bin/p/bin_elf.inc

```diff
@@ -1118,7 +1102,7 @@ static RBinInfo* info(RBinFile *bf) {
 	if (!(ret = R_NEW0 (RBinInfo))) {
 		return NULL;
 	}
-	ret->lang = is_cpp (bf) ? "c++" : "c";
+	ret->lang = r_bin_is_cxx (bf) ? "cxx" : "c";
```

Comment:
why did you change the string from `c++` to `cxx`? I'd keep it as it was if there is no particular reason.

Reply:
It s to be "in line" with the mangling feature change. But I can put it back if it triggers annoyance.

Reply:
I see